disperser/meterer/types.go

```diff
@@ -0,0 +1,243 @@
+package meterer
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"math/big"
+
+	"github.com/Layr-Labs/eigenda/core"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/common/math"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/signer/core/apitypes"
+)
+
+/* SUBJECT TO BIG MODIFICATIONS */
+
+// BlobQuorumParam represents the parameters for a specific quorum
+type BlobQuorumParam struct {
+	QuorumID           core.QuorumID
+	AdversaryThreshold uint32
+	QuorumThreshold    uint32
+}
+
+// BlobHeader represents the header information for a blob
+type BlobHeader struct {
+	// Existing fields
+	Commitment       core.G1Point
+	DataLength       uint32
+	BlobQuorumParams []BlobQuorumParam
+
+	// New fields
+	Version   uint32
+	AccountID string
+	Nonce     uint32 // use nonce to prevent duplicate payments in the same reservation window
+	BinIndex  uint64
+
+	Signature []byte
+	BlobSize  uint32
```

Comment:
This is redundant with DataLength. We should decide on canonical naming.

Reply:
went with `DataLength`